indexing-hadoop/src/main/java/io/druid/indexer/DetermineHashedPartitionsJob.java

```diff
@@ -105,7 +105,7 @@ public boolean run()
       }
       JobHelper.setupClasspath(
           JobHelper.distributedClassPath(config.getWorkingPath()),
-          JobHelper.distributedClassPath(config.makeIntermediatePath()),
+          JobHelper.distributedClassPath(new Path(config.makeIntermediatePath(), "groupBy")),
```

Comment:
can we replace "groupBY" by "determineHashedPartitionsJob" so that if someone looks directly at hdfs then its clear who owns the directory?